modules/ding_base/ding_base.module

```diff
@@ -535,3 +535,47 @@ function ding_base_rewrite_color_stylesheet() {
   variable_set('color_' . $theme . '_stylesheets', $css);
   variable_set('color_' . $theme . '_files', $paths['files']);
 }
+
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function ding_base_field_widget_form_alter(&$element, &$form_state, $context) {
+  $names = array(
+    'field_ding_page_title_image',
+    'field_ding_page_list_image',
+    'field_ding_tabroll_image',
+    'field_ding_news_title_image',
+    'field_ding_news_list_image',
+    'field_ding_library_title_image',
+    'field_ding_library_list_image',
+    'field_ding_group_title_image',
+    'field_ding_group_list_image',
+    'field_ding_event_title_image',
+    'field_ding_event_list_image'
+  );
+
+  if (isset($element['#field_name']) && in_array($element['#field_name'], $names)) {
```

Comment:
This change seems very helpful but is the predefined list of field names necessary? In other words: Could we apply this change for all image fields which have necessary configuration?